src/app/lists/ListDataview.tsx

```diff
@@ -19,49 +14,81 @@ import TooltipCell from "@/app/lists/TooltipCell";
 import TableSurface from "@/components/Tables/TableSurface";
 import CommentBox from "@/app/lists/CommentBox";
 
-export type ListRow = Schema["lists"] & Datum;
+interface ListRow {
+    itemName: string;
+    "1": QuantityAndNotes;
+    "2": QuantityAndNotes;
+    "3": QuantityAndNotes;
+    "4": QuantityAndNotes;
+    "5": QuantityAndNotes;
+    "6": QuantityAndNotes;
+    "7": QuantityAndNotes;
+    "8": QuantityAndNotes;
+    "9": QuantityAndNotes;
+    "10": QuantityAndNotes;
+}
+
+interface QuantityAndNotes {
+    quantity: string;
+    notes: string | null;
+}
 
 interface Props {
-    data: ListRow[];
+    data: Schema["lists"][];
     comment: string;
 }
 
-export const headerKeysAndLabels: [string, string][] = [
-    ["item_name", "Description"],
-    ["1_quantity", "Single"],
-    ["2_quantity", "Family of 2"],
-    ["3_quantity", "Family of 3"],
-    ["4_quantity", "Family of 4"],
-    ["5_quantity", "Family of 5"],
-    ["6_quantity", "Family of 6"],
-    ["7_quantity", "Family of 7"],
-    ["8_quantity", "Family of 8"],
-    ["9_quantity", "Family of 9"],
-    ["10_quantity", "Family of 10+"],
-];
-
-const listDataviewColumnDisplayFunctions: TableColumnDisplayFunctions = {};
-const listsColumnStyleOptions: TableColumnStyleOptions = {};
-
-listQuantityNoteAndLabels.forEach(([headerKey, noteKey]) => {
-    if (headerKey.endsWith("quantity")) {
-        // Column Display Function
-        listDataviewColumnDisplayFunctions[headerKey] = (row: Row) => {
-            const data = row.data as ListRow;
-            return (
-                <TooltipCell
-                    cellValue={data[headerKey]?.toString() ?? ""}
-                    tooltipValue={data[noteKey]?.toString() ?? ""}
-                />
-            );
-        };
-        // Column Style Function
-        listsColumnStyleOptions[headerKey] = {
-            minWidth: "9rem",
-            center: true,
-        };
-    }
-});
+export const headerKeysAndLabels = [
```

Comment:
The original one has a type `[string, string][]` ? Why is it not needed now

Reply:
That would be the wrong type as this is now [keyof ListRow, string]. Have a read of https://blog.logrocket.com/complete-guide-const-assertions-typescript/ if you're unsure of what the `as const` does here.